modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/DefinitionInfo/VersionRow.tsx

```diff
@@ -161,7 +161,9 @@ export function VersionRow({
 								userTimeZone
 							)}{' '}
 
-							by {creatorName}
+							{Liferay.Language.get('by').toLowerCase()}{' '}
+
+							{creatorName}
```

Comment:
here you could do `{' ' + creatorName}`

Reply:
Hey Rebecca, I ended up choosing to use Template Literals here. Let me know if everything is ok.